tests/test_use_flake.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
```

Comment:
Do we still need this? We don't need to support python 3.7 anymore for our internal test suite.